### DISCUSSION TITLE
Can replit.com be used to host pocketbase?

### DISCUSSION THREAD

**Discussion by jossmanbr2022** [pocketbase/pocketbase#353] [Q&A]
My doubt is if there is a way to host pocketbase, and that I can update the new versions of pocketbase easily, since I am a novice and I am just learning as I go, I would greatly appreciate help.

**Comment by ganigeorgiev** [comment]
While it may be possible to start a vm using replit I don't think it is a good idea to host a production app with it.
The free plan doesn't seem to include a private instance (or at least I couldn't find how with my quick search), so practically your files will be accessible by everyone.

Continuing https://github.com/pocketbase/pocketbase/discussions/328, I think [fly.io](https://fly.io/) would be better if you are looking for free hosting. I'll try to squeeze some time after v0.6.0 release and will create a simplified example(s) how to deploy on fly.io.
